pkg/Directory.Build.targets

```diff
@@ -0,0 +1,8 @@
+<Project>
+  <Import Project="..\Directory.Build.targets" />
+
+  <PropertyGroup Condition="'$(IsPackable)' != 'false'">
```

Comment:
This is done in a targets file rather than props to honor projects setting for IsPackable.